README.md

```diff
@@ -1 +1,44 @@
+## API
 
+### new PCR(x, y, intercept, weight)
```

Comment:
Please implement this as an options object. For example: `new PCR(x, y, { intercept: true, weight: 0.5 })`.